rustutils/src/lib.rs

```diff
@@ -0,0 +1 @@
+mod color;
```

Comment:
Needed to avoid link errors.

Reply:
Yeah, that's a standard Rust thing ;) Module files don't exist to the compiler unless they are declared.